lib/composer.js

```diff
@@ -194,6 +194,10 @@ export default class Composer extends Disposable {
 
     await Promise.all(jobs)
 
+    if (this.shouldCleanAfterBuild()) {
+      this.clean()
```

Comment:
This needs to be `await this.clean()` since `clean` is `async`.

We probably need to add a boolean parameter to clean specifying whether `latex.log.group()` and `latex.log.groupEnd()` need to be called. Currently this call to clean will fry any warnings/errors generated by the build because of the embedded calls to to `latex.log.group`.